packages/botonic-core/src/data-provider/data-provider.ts

```diff
@@ -1,13 +1,4 @@
-import { BotonicEvent } from '@botonic/core/lib/esm/models/events'
-import { User } from '@botonic/core/lib/esm/models/user'
-
-import { DynamoDBDataProvider } from './dynamodb-data-provider'
-import { LocalDevDataProvider } from './local-data-provider'
-
-export const dataProviderProtocols = {
-  DYNAMO_DB: 'dynamodb',
-  FILE: 'file',
-}
+import { BotonicEvent, User } from '../models'
 
 export interface DataProvider {
   getUsers(limit?: number, offset?: number): User[] | Promise<User[]>
```

Comment:
Does it make sense to have this file with only the interface defined? Would it be better to define this interface in `data-provider/index.t`s file or `data-provider/factory.ts` file?

Reply:
yes, maybe it's not that necessary. moved into factory.ts! (I preferred not to move it into index to preserve the same project structure -exporting all members in index- and avoid future problems with circulars deps)

Reply:
https://github.com/hubtype/botonic/pull/1866/commits/e6729b3139036a4ab1d66c8c7e8c103868883be3